routers/web/feed/release.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
```

Comment:
```suggestion
// SPDX-License-Identifier: MIT
```

Different headers were set in #21840